ArcGISRuntimeSDKQt_CppSamples/Scenes/ExtrudeGraphics/ExtrudeGraphics.cpp

```diff
@@ -97,7 +97,7 @@ void ExtrudeGraphics::componentComplete()
   graphicsOverlay->setRenderer(renderer);
 
   // setup graphic locations
-  QList<Point> pointsList;
+  QVector<Point> pointsList;
```

Comment:
In Qt6, this is not important, but in Qt5, we should use vector as default. (Use `std::vector` here?)